docs/tutorials/adapters.md

```diff
@@ -2,82 +2,3 @@
 
 OpenTimelineIO supports, or plans to support, conversion adapters for many
 existing file formats.
```

Comment:
There's not much left of this file.. do we still need it or should we at least flesh it out a bit?

Perhaps add links to each adapter repo?

Reply:
I re-wrote this to describe the current adapter landscape and to point readers to the plugin development tutorial.